ui/apps/platform/src/Containers/Risk/CreatePolicyFromSearch.tsx

```diff
@@ -50,7 +69,7 @@ function CreatePolicyFromSearch({
                     addFormMessage({ type: 'warn', message });
                 }
 
-                if (response?.hasNestedField) {
```

Comment:
Verified our front end proto against the backend proto. The field access here was just wrong.